pkg/content/cache.go

```diff
@@ -138,46 +138,43 @@ type RemoteCache struct {
 }
 
 func NewRemoteCache(cacheSize int, host remote.HostFunc) (*RemoteCache, error) {
-	remoteCache, err := lru.New[string, ocispec.Descriptor](cacheSize)
-	if err != nil {
-		return nil, err
-	}
 	return &RemoteCache{
-		remoteCache: remoteCache,
+		remoteCache: make(map[string]ocispec.Descriptor),
 		host:        host,
 		cacheSize:   cacheSize,
 	}, nil
 }
 
 func (rc *RemoteCache) Values() []ocispec.Descriptor {
-	return rc.remoteCache.Values()
+	var values []ocispec.Descriptor
+	for _, desc := range rc.remoteCache {
+		values = append(values, desc)
+	}
+	return values
 }
 
 func (rc *RemoteCache) Get(key string) (ocispec.Descriptor, bool) {
-	return rc.remoteCache.Get(key)
+	value, ok := rc.remoteCache[key]
+	return value, ok
 }
 
 func (rc *RemoteCache) Add(key string, value ocispec.Descriptor) {
-	rc.remoteCache.Add(key, value)
+	rc.remoteCache[key] = value
 }
 
 func (rc *RemoteCache) Remove(key string) {
-	rc.remoteCache.Remove(key)
+	delete(rc.remoteCache, key)
 }
 
-// Size returns the number of items in the cache.
 func (rc *RemoteCache) Size() int {
-	return rc.remoteCache.Len()
-
+	return len(rc.remoteCache)
 }
 
 func (rc *RemoteCache) NewLRUCache(cacheSize int, cacheRef string) error {
```

Comment:
Does this still be used?

Reply:
Actually not, maybe remove unused `Size()` and `Remove()`